sharding/receipt_consuming_tx_utils.py

```diff
@@ -77,6 +93,8 @@ def send_msg_add_used_receipt(state, shard_id, receipt_id):
 
 
 def send_msg_transfer_value(mainchain_state, shard_state, shard_id, tx):
+    assert validate_receipt_consuming_tx(mainchain_state, shard_state, shard_id, tx)
```

Comment:
IMO, the `assert` should only be used in the scenario that an event should always be `True` or `False` in our program.
For example, `validate_receipt_consuming_tx` might throw some exception like `InvalidTransaction` when we receive an ill-formed transaction from the network.

Reply:
Ahhh, sorry, it's a careless mistake.
It is supposed to be only:
```
def send_msg_transfer_value(mainchain_state, shard_state, shard_id, tx):
    validate_receipt_consuming_tx(mainchain_state, shard_state, shard_id, tx)
```
And the exceptions from `add_shard_transaction` from `apply_collation` from `add_collation` will be caught.

Reply:
Get it. Thank you:)